keyboards/nullbitsco/nibble/config.h

```diff
@@ -0,0 +1,63 @@
+/* Copyright 2018 Jack Humbert
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#include "config_common.h"
+
+/* Used to set master for remote KB if VUSB detect doesn't work. */
+// #define KEYBOARD_MASTER
+#define NO_USB_STARTUP_CHECK
```

Comment:
Is this needed?

Reply:
Might be needed for MacOS wake from sleep? I vaguely recall reading that somewhere.

Reply:
@nblyumberg I apologize, I let this one slip through the cracks. You're exactly right; I followed the suggestion [here](https://github.com/qmk/qmk_firmware/issues/5585#issuecomment-491255880). I followed the rest of the thread but was still seeing the issue with the latest QMK.

Reply:
To be honest/blunt, any info from that thread wouldn't apply to this board, as the issue there is with chibiOS, and therefore ARM boards. 

This board appears to be an AVR board, and not running any chibiOS related code.

Reply:
@drashna Got it, I skimmed that thread and didn't realize the HS60 was an ARM board. It is an issue I've seen in the wild (including repro on my own keeb) and confirmed it was resolved by adding `NO_USB_STARTUP_CHECK`. I'd prefer it stay in since most users are just flashing via QMK configuratior or toolbox and aren't building a binary themselves.

If it's an absolute no-go and removing it will get this PR through, then so be it. I'll add a note to the readme either way.